client/flutter/test/home_page_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:WHOFlutter/constants.dart';
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/home_page.dart';
+
+void main() {
+  Widget testableWidget({Widget child}) {
+    return MaterialApp(
+      title: "WHO",
+      localizationsDelegates: [
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        S.delegate
+      ],
+      locale: Locale('en', ''),
+      supportedLocales: S.delegate.supportedLocales,
+      theme: ThemeData(
```

Comment:
I believe setting `theme` for this particular test case is redundant.

Reply:
Yes, actually I was thinking to add test to check if the theme is applied properly or not. But it won't add any value right now. Thanks for pointing out. I have removed the theme in the next commit